README.md

```diff
@@ -42,17 +42,20 @@ The Bond repository uses Git submodules and should be cloned with the
 git clone --recursive https://github.com/Microsoft/bond.git
 ```
 
-In order to build Bond you will need CMake (3.1+), [Haskell Stack](https://docs.haskellstack.org/en/stable/README/#how-to-install) and
-Boost (1.58+). Bond's C++ library requires some C++11 features (currently
-limited to those supported bv Visual C++ 2013). (Note: Boost 1.59 may not
-work with Bond Comm due to some bugs in that version of the Boost ASIO library).
+Bond C++ library requires some C++11 features (currently limited to those
+supported bv Visual C++ 2013); a C++11 compiler is required. Additionall, to
+build Bond you will need CMake (3.1+),
+[Haskell Stack](https://docs.haskellstack.org/en/stable/README/#how-to-install)
+and Boost (1.58+). (Note: Boost 1.59 may not work with Bond Comm due to some
+bugs in that version of the Boost ASIO library).
 
 Following are specific instructions for building on various platforms.
 
 ### Linux
 
-Bond can be built with Clang (3.4+) or GNU C++ (4.7+). We recommend the latest
-version of Clang as it's much faster with template-heavy code like Bond.
+Bond can be built with Clang (3.4+) or GNU C++ (4.7+). We recommend the
```

Comment:
Talked with @ara-ayvazyan - we may still build with `gcc` 4.7, but we now require a `libstdc++` that normally ships with 5.4. Let's bump this version while we're in here. We might also want to bump `clang` to 3.8.

Reply:
Looking into this...

Reply:
Reworded.